cmd/kubeadm/app/phases/bootstraptoken/node/tlsbootstrap.go

```diff
@@ -88,3 +90,27 @@ func AutoApproveNodeBootstrapTokens(client clientset.Interface, k8sVersion *vers
 		},
 	})
 }
+
+// AutoApproveNodeCertificateRotation creates RBAC rules in a way that makes Node certificate rotation CSR auto-approved by the csrapprover controller
+func AutoApproveNodeCertificateRotation(client clientset.Interface) error {
+
+	fmt.Println("[bootstraptoken] Configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node certificate rotation")
+
+	// Always create this kubeadm-specific binding though
+	return apiclient.CreateOrUpdateClusterRoleBinding(client, &rbac.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: NodeAutoApproveCertificateRotationClusterRoleBinding,
+		},
+		RoleRef: rbac.RoleRef{
+			APIGroup: rbac.GroupName,
+			Kind:     "ClusterRole",
+			Name:     CSRAutoApprovalClusterRoleName,
```

Comment:
This is not accurate, should point to `...:selfnodeclient`, not `nodeclient`

Reply:
This would mean that for 1.8 cherry-pick we will need to do another patch where it will have similar to 1.8's AutoApproveNodeBootstrapTokens() functionality to make sure that this role exists. Right ?